include/action.h

```diff
@@ -2,19 +2,48 @@
 #ifndef __LABWC_ACTION_H
 #define __LABWC_ACTION_H
 
-struct server;
+/* Forward declare arguments */
```

Comment:
Consider "/* Forward declaration of opaque structs for pointer arguments */"
Although I think I prefer leaving it out because we should try to do that wherever we can.

Reply:
Will remove it completely then